draft-js-mention-plugin/src/MentionSuggestions/index.js

```diff
@@ -256,6 +255,24 @@ export default class MentionSuggestions extends Component {
     }
   };
 
+  renderSuggestion = (mention, index) => {
+    const { theme = {} } = this.props;
+    const key = genKey();
```

Comment:
Yeah, you're right, I've been a little quick on that one. I'll add a key generator option with a default of `mention.get('name')` and fix the missing id ;)